frameworks/g-cloud-9/questions/services/dataProtectionWithinSuppliersNetwork.yml

```diff
@@ -1,20 +1,20 @@
-name: "Data protection within the supplier’s network"
-question: "How do you protect data within your network?"
-optional: true
+name: Data protection within the supplier’s network
+question: How do you protect data within your network?
+
 depends:
-  -
-    "on": lot
+  - "on": lot
     being:
       - cloud-hosting
       - cloud-software
 
 type: checkboxes
 options:
-    -
-        label: "Private WAN service"
-    -
-        label: "TLS (Version 1.2 or above)"
-    -
-        label: "IPsec or TLS VPN gateway"
-    -
-        label: "Legacy SSL and TLS (under 1.2)"
+  - label: Private WAN service
+  - label: TLS (Version 1.2 or above)
+  - label: IPsec or TLS VPN gateway
+  - label: Legacy SSL and TLS (under 1.2)
+  - label: Other
```

Comment:
Yeah. We have 'Describe how else you protect data between the buyer’s network and your network.' as a follow up to users choosing other. Otherwise we would have to make the question optional (remember our discussion about checkboxes behaving like buttons). We could wrap it in 'Do you protect data within your network?'.